src/Components/DonateLevelCard.jsx

```diff
@@ -15,25 +15,23 @@ const useStyles = makeStyles(theme => ({
     },
 }));
 
-const DonateLevelCard = ({ amount, image, text, alt }) => {
+function DonateLevelCard({ amount, image, text, alt }) {
```

Comment:
Is there a reason we want to move away from arrow functions?

Reply:
I also prefer arrow functions 👍

Reply:
I see. I think I'd prefer to use function components that still embrace arrow functions and be using ES6 styntax